dwave/plugins/torch/autoencoder/losses/kl_divergence.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2025 D-Wave
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from dimod import Sampler
+
+from dwave.plugins.torch.boltzmann_machine import AbstractBoltzmannMachine
+
+__all__ = ["pseudo_kl_divergence_loss"]
+
+binary_cross_entropy_with_logits = torch.nn.BCEWithLogitsLoss()
+
+
+def pseudo_kl_divergence_loss(
+    spins: torch.Tensor,
+    logits: torch.Tensor,
+    boltzmann_machine: AbstractBoltzmannMachine,
+    sampler: Sampler,
+    sampler_kwargs: dict,
+):
+    """
+    A pseudo Kullback-Leibler divergence loss function for a Boltzmann machine. This is
```

Comment:
should this read "loss function for DVAE with BM prior" instead?

Reply:
Yes, I think that's better.